astropy/wcs/wcsapi/fitswcs.py

```diff
@@ -0,0 +1,282 @@
+# This file includes the definition of a mix-in class that provides the low-
+# and high-level WCS API to the astropy.wcs.WCS object. We keep this code
+# isolated in this mix-in class to avoid making the main wcs.py file too
+# long.
+
+import warnings
+
+import numpy as np
+
+from ... import units as u
+
+from .low_level_api import BaseLowLevelWCS
+from .high_level_api import HighLevelWCSMixin
+
+__all__ = ['custom_ctype_to_ucd_mapping', 'FITSWCSAPIMixin']
+
+# Mapping from CTYPE axis name to UCD1
+
+CTYPE_TO_UCD1 = {
+
+    # Celestial coordinates
+    'RA': 'pos.eq.ra',
+    'DEC': 'pos.eq.dec',
+    'GLON': 'pos.galactic.lon',
+    'GLAT': 'pos.galactic.lat',
+    'ELON': 'pos.ecliptic.lon',
+    'ELAT': 'pos.ecliptic.lat',
+    'TLON': 'pos.bodyrc.lon',
+    'TLAT': 'pos.bodyrc.lat',
+    'HPLT': 'custom:pos.helioprojective.lat',
+    'HPLN': 'custom:pos.helioprojective.lon',
+
+    # Spectral coordinates (WCS paper 3)
+    'FREQ': 'em.freq',  # Frequency
+    'ENER': 'em.energy',  # Energy
+    'WAVN': 'em.wavenumber',  # Wavenumber
+    'WAVE': 'em.wl',  # Vacuum wavelength
+    'VRAD': 'spect.dopplerVeloc.radio',  # Radio velocity
+    'VOPT': 'spect.dopplerVeloc.opt',  # Optical velocity
+    'ZOPT': 'src.redshift',  # Redshift
+    'AWAV': 'em.wl',  # Air wavelength
+    'VELO': 'spect.dopplerVeloc',  # Apparent radial velocity
+    'BETA': None,  # Beta factor (v/c)
```

Comment:
why not a "custom" here? Maybe "custom:spect.doplerVeloc.beta" or similar?

Reply:
Done. I've also emailed the VO people to ask for a proper term.